src/frontend/apps/e2e/__tests__/app-desk/menu.spec.ts

```diff
@@ -58,19 +58,22 @@ test.describe('Menu', () => {
 
       // eslint-disable-next-line playwright/no-conditional-in-test
       if (isDefault) {
+        // eslint-disable-next-line playwright/no-conditional-expect
         await expect(
           page.getByRole('button', {
             name: 'Create a new team',
           }),
         ).toBeVisible();
       } else {
+        // eslint-disable-next-line playwright/no-conditional-expect
         await expect(
           page.getByRole('button', {
             name: 'Create a new team',
           }),
         ).toBeHidden();
 
         const reg = new RegExp(name.toLowerCase());
+        // eslint-disable-next-line playwright/no-conditional-expect
         await expect(page).toHaveURL(reg);
       }
```

Comment:
nit: you could use ablock-comment to disable eslint (cf [here](https://eslint.org/docs/latest/use/configure/rules#using-configuration-comments-1)).